lib/grp.gi

```diff
@@ -1342,6 +1342,50 @@ end);
 ##
 #M  MaximalSubgroups( <G> )
 ##
+InstallMethod(MaximalSubgroupClassReps,"default, catch dangerous options",
```

Comment:
Should there perhaps also be a `TryMaximalSubgroupClassReps` method in the filter `HasMaximalSubgroupClassReps`, like this:
```gap
InstallMethod(TryMaximalSubgroupClassReps,
    [IsGroup and HasMaximalSubgroupClassReps],
    MaximalSubgroupClassReps);
```
Possibly with a high rank adjustment.

Reply:
Will certainly not harm. Added.